wasmsdk/jsbridge/func_test.go

```diff
@@ -0,0 +1,87 @@
+// +build js,wasm
```

Comment:
Last time I check, adding test file with `+build js,wasm` make the output file bigger. Please double check it again. If it is the case, then I propose not include the test file to the final build.

Reply:
tested it. they seems be same size. because test files are not built on `go build`. they work with `go test` only.